hexrd/material.py

```diff
@@ -377,6 +378,26 @@ def update_structure_factor(self):
 
         self.planeData.set_structFact(sf)
 
+    def compute_powder_overlay(self,
+                               ttharray=numpy.linspace(0, 80, 2000),
```

Comment:
It looks like this isn't causing an issue here, but for future reference, it's best practice to not have any mutable default arguments, since it can cause hard to track down bugs. Integers, floats, strings, etc. are all immutable, but a numpy array is mutable. Here's an example:

```python
import numpy as np
  

def example(a=np.linspace(1, 2, 2)):
    return a

out = example()
print(f'{out=}')  # out=array([1., 2.])
out[:] = 0
out2 = example()
print(f'{out2=}')  # out2=array([0., 0.])
```

You can see that since the numpy array was modified, the default argument was also modified.

The better way to do this is to default `ttharray=None`, and then in the function:
```python
if ttharray is None:
    ttharray = numpy.linspace(0, 80, 2000)
```